install.sh

```diff
@@ -1,354 +1,287 @@
+
+FPM_FC=`which $MPIFC`
+FPM_CC=`which $MPICC`
+
+#echo "# DO NOT EDIT OR COMMIT -- Created by opencoarrays/install.sh" > build/fpm.toml
+#cp manifest/fpm.toml.template build/fpm.toml
+#FPM_TOML_LINK_ENTRY="link = [\"$(echo ${GASNET_LIB_NAMES} | sed 's/ /", "/g')\"]"
+#echo "${FPM_TOML_LINK_ENTRY}" >> build/fpm.toml
+#ln -f -s build/fpm.toml
+#
+#OPENCOARRAYS_PC="$PREFIX/lib/pkgconfig/opencoarrays.pc"
+#echo "OPENCOARRAYS_FPM_LDFLAGS=$GASNET_LDFLAGS $GASNET_LIB_LOCATIONS"             >  $OPENCOARRAYS_PC
+#echo "OPENCOARRAYS_FPM_FC=$FPM_FC"                                                >> $OPENCOARRAYS_PC
+#echo "OPENCOARRAYS_FPM_FLAGS=-fcoarray=lib"                                       >> $OPENCOARRAYS_PC
+#echo "OPENCOARRAYS_FPM_CC=$FPM_CC"                                                >> $OPENCOARRAYS_PC
+#echo "OPENCOARRAYS_FPM_CFLAGS=-DPREFIX_NAME=_gfortran_caf_ -DGCC_GE_7 -DGCC_GE_8" >> $OPENCOARRAYS_PC
+#echo "Name: OpenCoarrays"                                                         >> $OPENCOARRAYS_PC
+#echo "Description: Coarray Fortran parallel runtime library"                      >> $OPENCOARRAYS_PC
+#echo "URL: https://github.com/sourceryinstitute/opencoarrays"                     >> $OPENCOARRAYS_PC
+#echo "Version: 3.0.0"                                                             >> $OPENCOARRAYS_PC
+#
+#exit_if_pkg_config_pc_file_missing()
+#{
+#  if ! $PKG_CONFIG $1 ; then
+#    echo "$1.pc pkg-config file not found"
+#    exit 1
+#  fi
+#}
+#exit_if_pkg_config_pc_file_missing "opencoarrays"
+#
+#RUN_FPM_SH="build/run-fpm.sh"
+#echo "#!/bin/sh"                                                                  >  $RUN_FPM_SH
+#echo "#-- DO NOT EDIT -- created by opencoarrays/install.sh"                      >> $RUN_FPM_SH
+#echo "\"${FPM}\" \"\$@\" \\"                                                      >> $RUN_FPM_SH
+#echo "--compiler \"`$PKG_CONFIG opencoarrays --variable=OPENCOARRAYS_FPM_FC`\"   \\"  >> $RUN_FPM_SH
+#echo "--c-compiler \"`$PKG_CONFIG opencoarrays --variable=OPENCOARRAYS_FPM_CC`\" \\"  >> $RUN_FPM_SH
+#echo "--c-flag \"`$PKG_CONFIG opencoarrays --variable=OPENCOARRAYS_FPM_CFLAGS`\" \\"  >> $RUN_FPM_SH
+#echo "--link-flag \"`$PKG_CONFIG opencoarrays --variable=OPENCOARRAYS_FPM_LDFLAGS`\"" >> $RUN_FPM_SH
+#chmod u+x $RUN_FPM_SH
+#
+#./$RUN_FPM_SH build
+
+$FPM install \
+  --compiler $FPM_FC \
+  --c-compiler $FPM_CC \
+  --c-flag "-DPREFIX_NAME=_gfortran_caf_ -DGCC_GE_7 -DGCC_GE_8" \
+  --flag "-fcoarray=lib"
+
+CAF_VERSION=$(sed -n '/[0-9]\{1,\}\(\.[0-9]\{1,\}\)\{1,\}/{s/^\([^.]*\)\([0-9]\{1,\}\(\.[0-9]\{1,\}\)\{1,\}\)\(.*\)/\2/p;q;}' .VERSION)
+MPIFORT_SHOW=$($MPIFC -show)
+Fortran_COMPILER="${MPIFORT_SHOW%% *}"
+CAF_MPI_LIBS="${MPIFORT_SHOW#* }"
+
+sed -e "s/@CAF_VERSION@/$CAF_VERSION/g" src/wrappers/caf.in > "$PREFIX"/bin/caf
+sed -i '' -e "s/@Fortran_COMPILER@/$Fortran_COMPILER/g" "$PREFIX"/bin/caf
+sed -i '' -e "s/@CAF_LIBS@/lib\/libopencoarrays.a/g" "$PREFIX"/bin/caf
```

Comment:
I'm pretty sure these `sed` commands have to have `-i''` (without the space) or just `-i` on Linux.